From 6976233c24367c799e84f376f8e6cbc2c55a698d Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Sat, 28 Jun 2014 17:06:48 -0700
Subject: [PATCH 14/16] Fix mytharchive compliation.

mythframe.h is used by mythavutil.h, so make it a public header
---
 mythtv/libs/libmythtv/libmythtv.pro |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/libmythtv.pro b/mythtv/libs/libmythtv/libmythtv.pro
index 1417b94..ea3df12 100644
--- a/mythtv/libs/libmythtv/libmythtv.pro
+++ b/mythtv/libs/libmythtv/libmythtv.pro
@@ -245,7 +245,8 @@ SOURCES += srtwriter.cpp
 
 inc.path = $${PREFIX}/include/mythtv/
 inc.files  = playgroup.h
-inc.files += mythtvexp.h            metadataimagehelper.h mythavutil.h
+inc.files += mythtvexp.h            metadataimagehelper.h
+inc.files += mythavutil.h           mythframe.h
 
 INSTALLS += inc
 
-- 
1.7.10.2

